hack/verify-vendor.sh

```diff
@@ -27,7 +27,7 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 # create a nice clean place to put our new vendor tree
 mkdir -p "${KUBE_ROOT}/_tmp"
-_tmpdir="$(mktemp -d "${KUBE_ROOT}/_tmp/kube-vendor.XXXXXX")"
+_tmpdir="$(kube::realpath "$(mktemp -d "${KUBE_ROOT}/_tmp/$(basename "$0").XXXXXX")")"
```

Comment:
This is repeated everywhere. Would it make sense to wrap it into a function, e.e. `kube::mktemp` or something similar?

Reply:
Make sense, I just followed @thockin 's instructions.